frontend/src/views/OrderView.vue

```diff
@@ -115,12 +116,56 @@ const removeOrder = async () => {
 const confirmDelete = () => {
   confirm.showDeleteOrderConfirmation(removeOrder);
 };
+
 const findClientById = () => {
   const foundClient = clientsList.value.find((client) => client.id === orderInfo.value.client_id);
 
   return foundClient;
 };
 
+const subItemsIds = computed(() => {
+  const result = [];
+
+  if ((orderInfo.value as Order).offer) {
+    result.push({
+      type: "OFFER",
```

Comment:
We have an enum for the type, no?

Reply:
But isn't "documentType" and "subitemKind" the same? 🤔 

In any case I can refactor this later 👌🏻